app/assets/javascripts/TortoiseJS/agent/view.coffee

```diff
@@ -248,17 +248,37 @@ class TurtleView extends View
     @ctx.lineTo(x2,y2)
 
   drawLink: (link, turtles, canWrapX, canWrapY) ->
+    shouldWrapInDim = (canWrap, worldMin, worldMax, cor1, cor2) ->
+      dist    = Math.abs(cor1 - cor2)
+      dimSize = worldMax - worldMin + 1
```

Comment:
Dimensions are stored as `@patchWidth` and `@patchHeight`. Should probably use those instead (so, take an argument `dimension` or something)...